packages/dsi-logo-maker/functions/textAndImageSVG/index.js

```diff
@@ -25,39 +27,41 @@ export const handler = ({ inputs, mechanic }) => {
   const words = text.split(" ").map((s) => s.toUpperCase());
   const colors = getColors("Custom Colors", null, colorsString.split(","));
   const height = Math.floor((width / ratio) * rows);
+  const font = useLoadedOpentypeFont(fontMode);
 
-  const position = { x: image ? height : 0, y: 0 };
-  const blockGeometry = computeBlockGeometry(width, height, rows, cols);
-  const baseBricks = computeBaseBricks(words, blockGeometry.fontSize);
-
-  const block = computeBlock(
-    blockGeometry,
-    baseBricks,
-    Math.floor(offset * baseBricks.length)
-  );
-
-  useEffect(() => {
-    if (image) {
+  useMemo(() => {
+    if (image && font) {
       const reader = new FileReader();
-
       reader.readAsDataURL(image);
-
       reader.onload = function () {
         setHref(reader.result);
       };
-
       reader.onerror = function () {
         console.error(reader.error);
       };
     }
-  }, []);
+  }, [font, image]);
```

Comment:
Same here, clicking the image input doesn't open a file picker. As mentioned before, probably unrelated to your changes.

I'll look into what's causing this and open an issue for it :blush:

Reply:
Done #148